src/app/components/quotes/quotes.hbs

```diff
@@ -1,36 +1,36 @@
 <style scoped>
-    @import url(/app/pages/home/quotes/quotes.css);
+    @import url(/app/components/quotes/quotes.css);
 </style>
 <div class="quotes">
     <div class="quote-bucket right">
-        <img src="/images/home/quotes/juan.jpg" width="175" height="230" />
+        <div class="image juan"></div>
```

Comment:
Yeah, I think each class will have to be specifically tied to each image.  So we'll need a:
```
.name {
    background-image: url(src);
```
for every single image to make it something that can be updated from the backend.  Other possible options include putting the style element directly on the div, but that has its own problems since it will likely mean mixing media-queries into the HTML too, which just starts getting really messy really fast.

I definitely don't like it, but I don't think we have another option for the time being.  If I'm missing a solution, please let me know.